paasta_tools/paasta_serviceinit.py

```diff
@@ -85,7 +87,16 @@ def main():
         sys.exit(1)
 
     cluster = load_system_paasta_config().get_cluster()
+    actual_deployments = get_actual_deployments(service, args.soa_dir)
+
     for instance in instances:
+        # For an instance, there might be multiple versions running, e.g. in crossover bouncing.
+        # In addition, mesos master does not have information of a chronos service's git hash.
+        # The git sha in deployment.json is simply used here.
+        version = actual_deployments['.'.join((cluster, instance))][:8]
```

Comment:
This is a bit odd, we don't normally report the git sha in this way.

I'm "ok" with this as a temporary measure, but longer term I would rather this be explicit about the desired versus actual here, because this reflects the "desired" git sha